File: demo/springboot/demo/src/main/java/com/demo/domain/entity/UserEntity.java
package com.demo.domain.entity;

import cn.hutool.core.util.IdUtil;
import lombok.Data;
import org.jetbrains.annotations.Nullable;
import org.springframework.data.annotation.Id;

@Data
public class UserEntity {
    @Id
    private Long id = IdUtil.getSnowflake().nextId();
    private String name;
    private String account;
    private String password;
    @Nullable
    private Long ossId = null;
}

File: demo/kmp/shared/src/commonMain/kotlin/com/vectordemo/domain/entity/UserEntity.kt
package com.vectordemo.domain.entity

data class UserEntity(
    val id: Long = 0L,
    val userId: Long,
    val account: String,
    val name: String,
    val avatarUrl: String,
    val accessToken: String,
    val password: String,
    val isCurrent: Boolean,
    val lastLoginAt: Long,
)
